utils/api_util.py

```diff
@@ -0,0 +1,111 @@
+"""Ingests historical CSV data into DB."""
+
+import json
+import os
+from datetime import datetime
```

Comment:
I've used `arrow` for datetime management throughout this codebase. If you continue to use that we can maintain logical consistency. It's simpler to use than the default `datetime` library.

Reply:
Sure then let me also use `arrow` to keep it consistent

Reply:
now not using datetime